libs/ha/ha_comp_libs.py

```diff
@@ -211,3 +211,26 @@ def get_ha_log_prop(node_obj, log_name: str, kvalue: int, fault_tolerance: bool
             resp_dict['node_id'].append(node_id)
             resp_dict['resource_id'].append(resource_id)
         return resp_dict
+
+    @staticmethod
+    def get_word_count(node_obj, log_index: int):
```

Comment:
Function name is too generic. As this function gets word count from ha logs only you need to rename it to something like get_ha_log_wc().

Reply:
Done